spec.html

```diff
@@ -32082,6 +32102,59 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
```

Comment:
This section replaces https://tc39.es/ecma402/#sec-time-zone-names and adds more clarification around terminology and concepts for time zone identifiers.

It also adds some non-normative recommendations for implementers, mostly recapitulating existing "it is recommended" text in 262 and/or 402, but adds more detail and clarity and provides recommendations for avoiding possible footguns.